E2E Failures on Develop

elipe17 opened the pull request:
## Summary of Changes
- Added appropriate cookie settings for all deployed app instances
- Updated the CypressLoginDotGovAuthenticationOverride class to behave like our other token auth endpoints (GET instead of POST)
- Updated Cypress user/admin login commands to adhere to the new GET endpoint for Cypress auth
- Updated submission history value check and left detailed comment as to why the change was made
Pull request closes #5495 

## How to Test
```
cd tdrs-frontend && docker-compose up --build
cd tdrs-backend && docker-compose up --build
```

1. Run the E2E tests locally and ensure they pass
2. Deploy this PR to the `raft` environment 
3. Grab the CYPRESS_TOKEN from the `tdp-backend-raft` environment variables
4. Set the CYPRESS_TOKEN in your shell and run the command below. Ensure all the tests pass.
```bash
npm run test:e2e-ci -- --config baseUrl="https://tdp-frontend-raft.app.cloud.gov" --env cypressToken=$CYPRESS_TOKEN,apiUrl="https://tdp-frontend-raft.app.cloud.gov/v1",adminUrl="https://tdp-frontend-raft.app.cloud.gov/admin"
```

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [ ] Cypress tests pass for deployed environments
+ [ ] Cyrpres tests pass locally

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [ ] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?


### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [ ] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [ ] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

elipe17 committed:
- Update auth_check to always return CSRF
- Fixed doc string
- Update login commands to call auth check first to get csrf and then hit the login endpoint

elipe17 committed:
- Add missing cookie settings to deployed settings config
- Remove csrf from auth check
- Update cypress auth endpoint to mirror real user auth
- Pass cypress token in custom token header
- Update cypress auth commands to use GET and pass appropriate headers and query params

elipe17 committed:
- remove extra comment

elipe17 committed:
- updated test validation criteria and documented why the change is ok

elipe17 committed:
- organize settings better

elipe17 committed:
Merge branch 'develop' into 5495-referer-e2e-failures

jtimpe commented:
i saw some failures testing locally, but i'm not sure what's related to the PR or not. it seems the datafile status endpoint returns a `502`, but looking at the logs, the only error i see is in the frontend
```bash
   2025-12-12T15:15:53.51-0600 [APP/PROC/WEB/0] ERR 2025/12/12 21:15:53 [error] 118#0: *1977 limiting requests, excess: 1.000 by zone "limitreqsbyaddr", client: 2a09:bac1:76c0:f50::5e:27, server: , request: "POST /v1/logs/ HTTP/1.1", host: "tdp-frontend-raft.app.cloud.gov", referrer: "https://tdp-frontend-raft.app.cloud.gov/data-files?fy=2021&q=Q1&type=tanf&tab=1"
```
which is for a different request entirely and doesn't show up in the failures. in fact, searching for the specific request, i only see a success (reported in both backend and frontend logs)
```
"GET /v1/data_files/954/ HTTP/1.1" 200 0 704
```
screenshot shows the different errors. some appear to be mismatched user permissions/locations for the test. i suppose the location IDs could be different in different envs, so maybe we need to `dumpdata --natural-foreign` like with user permissions.
<img width="1280" height="633" alt="Data file submission -- A FRA data analyst can submit a SSP data file (failed)" src="https://github.com/user-attachments/assets/8a85d9d1-c551-45ff-a17b-2c728565abc7" />

<img width="1280" height="633" alt="Data file submission -- A FRA data analyst can submit a Work Outcomes of TANF Exiters data file (failed)" src="https://github.com/user-attachments/assets/158ffb53-d84a-4137-acf4-554d1ce580ed" />

<img width="1280" height="633" alt="Data file submission -- A FRA data analyst sees incorrect file type error (failed)" src="https://github.com/user-attachments/assets/b816e8e2-8363-42eb-9bcd-20030e772092" />

<img width="1280" height="633" alt="Data file submission -- Admin Alex can submit FRA reports (failed)" src="https://github.com/user-attachments/assets/4b4a9c30-41f2-4e7e-887d-e4acecc3bce2" />

<img width="1280" height="633" alt="Data file submission -- Data Analyst Stefani can submit a SSP Section 3 data file (example #7) (failed)" src="https://github.com/user-attachments/assets/63e48e33-f458-45e8-91ea-d7894a9ac530" />

raftmsohani commented on tdrs-backend/tdpservice/users/api/login.py:
```diff
@@ -443,12 +446,12 @@ def get_auth_options(self, access_token, sub):
+    """Override Login.gov authentication for Cypress users."""
 
-    def post(self, request):
+    def get(self, request):
```
1) devLogin in (tdrs-frontend/src/index.js line 82) still POSTs  to /login/cypress
2) k6 login helper also POST

elipe17 replied:
I removed devLogin all together since none of us use it and updated the K6 tests to use the GET endpoint. Nice catches!

elipe17 committed:
- Remove devLogin period
- Update k6 tests to use GET endpoint

elipe17 commented:
> i saw some failures testing locally, but i'm not sure what's related to the PR or not. it seems the datafile status endpoint returns a `502`, but looking at the logs, the only error i see is in the frontend
> 
> ```shell
>    2025-12-12T15:15:53.51-0600 [APP/PROC/WEB/0] ERR 2025/12/12 21:15:53 [error] 118#0: *1977 limiting requests, excess: 1.000 by zone "limitreqsbyaddr", client: 2a09:bac1:76c0:f50::5e:27, server: , request: "POST /v1/logs/ HTTP/1.1", host: "tdp-frontend-raft.app.cloud.gov", referrer: "https://tdp-frontend-raft.app.cloud.gov/data-files?fy=2021&q=Q1&type=tanf&tab=1"
> ```
> 
> which is for a different request entirely and doesn't show up in the failures. in fact, searching for the specific request, i only see a success (reported in both backend and frontend logs)
> 
> ```
> "GET /v1/data_files/954/ HTTP/1.1" 200 0 704
> ```
> 
> screenshot shows the different errors. some appear to be mismatched user permissions/locations for the test. i suppose the location IDs could be different in different envs, so maybe we need to `dumpdata --natural-foreign` like with user permissions. <img alt="Data file submission -- A FRA data analyst can submit a SSP data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079055-8a85d9d1-c551-45ff-a17b-2c728565abc7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5MDU1LThhODVkOWQxLWM1NTEtNDVmZi1hMTdiLTJjNzI4NTY1YWJjNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZTZhN2Y4OWFjZmEyNGRkODFlMWVhOTY2OTZkNWRhYThmZDRkYzIzMzRiMzQ5Njc1ODIwZGQxOTFhYjY0NzUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FPLJKQOJMfFpNyJLqhxS34GiNf1z-zbLEipMqBZJgk4">
> 
> <img alt="Data file submission -- A FRA data analyst can submit a Work Outcomes of TANF Exiters data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079389-158ffb53-d84a-4137-acf4-554d1ce580ed.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5Mzg5LTE1OGZmYjUzLWQ4NGEtNDEzNy1hY2Y0LTU1NGQxY2U1ODBlZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjMwY2IyNDg4YzUyYjVlZTQ2M2YxZjhhMzQxMTFlYTY3MDczNTkwZTQzZTNhODE0MTU4NzFkZWQ1NGI1YWU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oX1BbW2EsZwaHVquTHSddS5YWpnqB_Lz0mjv7Ao1dTI"> <img alt="Data file submission -- A FRA data analyst sees incorrect file type error (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526080293-b816e8e2-8363-42eb-9bcd-20030e772092.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDgwMjkzLWI4MTZlOGUyLTgzNjMtNDJlYi05YmNkLTIwMDMwZTc3MjA5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMmZkNThkNTFjZTUzN2VjY2FjMTYyOGE2ZWU2MDNlYmRlZmRiMjIxYjNmNzkyMGM1YTk2Yzg0Y2Q5MjJjYWRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PSoqKg0GjL8OR8Abzb2eYUikF7cJvnrAvPuy04nKPAg"> <img alt="Data file submission -- Admin Alex can submit FRA reports (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079502-4b4a9c30-41f2-4e7e-887d-e4acecc3bce2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTAyLTRiNGE5YzMwLTQxZjItNGU3ZS04ODdkLWU0YWNlY2MzYmNlMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYzBlMTAwOGZhYzFjNWJlMDU4NTg0NWMyNDUzMGYyZWE1MmIyODM5ZTM0YTUxZDhlM2JkZjAwNTFkMTEzZjFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FnZ_POcqnP_HFZZ04i54VGAb12Fg9sz6lv6WTNGCxZQ"> <img alt="Data file submission -- Data Analyst Stefani can submit a SSP Section 3 data file (example #7) (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079530-63e48e33-f458-45e8-91ea-d7894a9ac530.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTMwLTYzZTQ4ZTMzLWY0NTgtNDVlOC05MWVhLWQ3ODk0YTlhYzUzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDllMDY5ODFkNzJkNGNmYTM5ZWU5MjY4OTQzNzk1MmVmZTBmNTJjNTBiMzE0MWI4YmEzZWVkNjY4NThkNTA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KSlzjpKx1Z1Blx9pk68HOAlR1hBgSoC1S_wZmJZsDnA">

@raftmsohani and @mattcoleanderson could you guys check and see if this is also happening for you?

elipe17 committed:
Merge branch 'develop' into 5495-referer-e2e-failures

raftmsohani commented:
> > i saw some failures testing locally, but i'm not sure what's related to the PR or not. it seems the datafile status endpoint returns a `502`, but looking at the logs, the only error i see is in the frontend
> > ```shell
> >    2025-12-12T15:15:53.51-0600 [APP/PROC/WEB/0] ERR 2025/12/12 21:15:53 [error] 118#0: *1977 limiting requests, excess: 1.000 by zone "limitreqsbyaddr", client: 2a09:bac1:76c0:f50::5e:27, server: , request: "POST /v1/logs/ HTTP/1.1", host: "tdp-frontend-raft.app.cloud.gov", referrer: "https://tdp-frontend-raft.app.cloud.gov/data-files?fy=2021&q=Q1&type=tanf&tab=1"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > which is for a different request entirely and doesn't show up in the failures. in fact, searching for the specific request, i only see a success (reported in both backend and frontend logs)
> > ```
> > "GET /v1/data_files/954/ HTTP/1.1" 200 0 704
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > screenshot shows the different errors. some appear to be mismatched user permissions/locations for the test. i suppose the location IDs could be different in different envs, so maybe we need to `dumpdata --natural-foreign` like with user permissions. <img alt="Data file submission -- A FRA data analyst can submit a SSP data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079055-8a85d9d1-c551-45ff-a17b-2c728565abc7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5MDU1LThhODVkOWQxLWM1NTEtNDVmZi1hMTdiLTJjNzI4NTY1YWJjNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZTZhN2Y4OWFjZmEyNGRkODFlMWVhOTY2OTZkNWRhYThmZDRkYzIzMzRiMzQ5Njc1ODIwZGQxOTFhYjY0NzUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FPLJKQOJMfFpNyJLqhxS34GiNf1z-zbLEipMqBZJgk4">
> > <img alt="Data file submission -- A FRA data analyst can submit a Work Outcomes of TANF Exiters data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079389-158ffb53-d84a-4137-acf4-554d1ce580ed.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5Mzg5LTE1OGZmYjUzLWQ4NGEtNDEzNy1hY2Y0LTU1NGQxY2U1ODBlZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjMwY2IyNDg4YzUyYjVlZTQ2M2YxZjhhMzQxMTFlYTY3MDczNTkwZTQzZTNhODE0MTU4NzFkZWQ1NGI1YWU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oX1BbW2EsZwaHVquTHSddS5YWpnqB_Lz0mjv7Ao1dTI"> <img alt="Data file submission -- A FRA data analyst sees incorrect file type error (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526080293-b816e8e2-8363-42eb-9bcd-20030e772092.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDgwMjkzLWI4MTZlOGUyLTgzNjMtNDJlYi05YmNkLTIwMDMwZTc3MjA5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMmZkNThkNTFjZTUzN2VjY2FjMTYyOGE2ZWU2MDNlYmRlZmRiMjIxYjNmNzkyMGM1YTk2Yzg0Y2Q5MjJjYWRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PSoqKg0GjL8OR8Abzb2eYUikF7cJvnrAvPuy04nKPAg"> <img alt="Data file submission -- Admin Alex can submit FRA reports (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079502-4b4a9c30-41f2-4e7e-887d-e4acecc3bce2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTAyLTRiNGE5YzMwLTQxZjItNGU3ZS04ODdkLWU0YWNlY2MzYmNlMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYzBlMTAwOGZhYzFjNWJlMDU4NTg0NWMyNDUzMGYyZWE1MmIyODM5ZTM0YTUxZDhlM2JkZjAwNTFkMTEzZjFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FnZ_POcqnP_HFZZ04i54VGAb12Fg9sz6lv6WTNGCxZQ"> <img alt="Data file submission -- Data Analyst Stefani can submit a SSP Section 3 data file (example #7) (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079530-63e48e33-f458-45e8-91ea-d7894a9ac530.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTMwLTYzZTQ4ZTMzLWY0NTgtNDVlOC05MWVhLWQ3ODk0YTlhYzUzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDllMDY5ODFkNzJkNGNmYTM5ZWU5MjY4OTQzNzk1MmVmZTBmNTJjNTBiMzE0MWI4YmEzZWVkNjY4NThkNTA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KSlzjpKx1Z1Blx9pk68HOAlR1hBgSoC1S_wZmJZsDnA">
> 
> @raftmsohani and @mattcoleanderson could you guys check and see if this is also happening for you?

@elipe17 I don't see any of the images and screenshots.

elipe17 commented:
> > > i saw some failures testing locally, but i'm not sure what's related to the PR or not. it seems the datafile status endpoint returns a `502`, but looking at the logs, the only error i see is in the frontend
> > > ```shell
> > >    2025-12-12T15:15:53.51-0600 [APP/PROC/WEB/0] ERR 2025/12/12 21:15:53 [error] 118#0: *1977 limiting requests, excess: 1.000 by zone "limitreqsbyaddr", client: 2a09:bac1:76c0:f50::5e:27, server: , request: "POST /v1/logs/ HTTP/1.1", host: "tdp-frontend-raft.app.cloud.gov", referrer: "https://tdp-frontend-raft.app.cloud.gov/data-files?fy=2021&q=Q1&type=tanf&tab=1"
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > which is for a different request entirely and doesn't show up in the failures. in fact, searching for the specific request, i only see a success (reported in both backend and frontend logs)
> > > ```
> > > "GET /v1/data_files/954/ HTTP/1.1" 200 0 704
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > screenshot shows the different errors. some appear to be mismatched user permissions/locations for the test. i suppose the location IDs could be different in different envs, so maybe we need to `dumpdata --natural-foreign` like with user permissions. <img alt="Data file submission -- A FRA data analyst can submit a SSP data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079055-8a85d9d1-c551-45ff-a17b-2c728565abc7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5MDU1LThhODVkOWQxLWM1NTEtNDVmZi1hMTdiLTJjNzI4NTY1YWJjNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZTZhN2Y4OWFjZmEyNGRkODFlMWVhOTY2OTZkNWRhYThmZDRkYzIzMzRiMzQ5Njc1ODIwZGQxOTFhYjY0NzUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FPLJKQOJMfFpNyJLqhxS34GiNf1z-zbLEipMqBZJgk4">
> > > <img alt="Data file submission -- A FRA data analyst can submit a Work Outcomes of TANF Exiters data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079389-158ffb53-d84a-4137-acf4-554d1ce580ed.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5Mzg5LTE1OGZmYjUzLWQ4NGEtNDEzNy1hY2Y0LTU1NGQxY2U1ODBlZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjMwY2IyNDg4YzUyYjVlZTQ2M2YxZjhhMzQxMTFlYTY3MDczNTkwZTQzZTNhODE0MTU4NzFkZWQ1NGI1YWU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oX1BbW2EsZwaHVquTHSddS5YWpnqB_Lz0mjv7Ao1dTI"> <img alt="Data file submission -- A FRA data analyst sees incorrect file type error (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526080293-b816e8e2-8363-42eb-9bcd-20030e772092.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDgwMjkzLWI4MTZlOGUyLTgzNjMtNDJlYi05YmNkLTIwMDMwZTc3MjA5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMmZkNThkNTFjZTUzN2VjY2FjMTYyOGE2ZWU2MDNlYmRlZmRiMjIxYjNmNzkyMGM1YTk2Yzg0Y2Q5MjJjYWRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PSoqKg0GjL8OR8Abzb2eYUikF7cJvnrAvPuy04nKPAg"> <img alt="Data file submission -- Admin Alex can submit FRA reports (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079502-4b4a9c30-41f2-4e7e-887d-e4acecc3bce2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTAyLTRiNGE5YzMwLTQxZjItNGU3ZS04ODdkLWU0YWNlY2MzYmNlMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYzBlMTAwOGZhYzFjNWJlMDU4NTg0NWMyNDUzMGYyZWE1MmIyODM5ZTM0YTUxZDhlM2JkZjAwNTFkMTEzZjFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FnZ_POcqnP_HFZZ04i54VGAb12Fg9sz6lv6WTNGCxZQ"> <img alt="Data file submission -- Data Analyst Stefani can submit a SSP Section 3 data file (example #7) (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079530-63e48e33-f458-45e8-91ea-d7894a9ac530.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTMwLTYzZTQ4ZTMzLWY0NTgtNDVlOC05MWVhLWQ3ODk0YTlhYzUzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDllMDY5ODFkNzJkNGNmYTM5ZWU5MjY4OTQzNzk1MmVmZTBmNTJjNTBiMzE0MWI4YmEzZWVkNjY4NThkNTA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KSlzjpKx1Z1Blx9pk68HOAlR1hBgSoC1S_wZmJZsDnA">
> > 
> > 
> > @raftmsohani and @mattcoleanderson could you guys check and see if this is also happening for you?
> 
> @elipe17 I don't see any of the images and screenshots.

@raftmsohani are you able to successfully run the e2e test suite? Jan was getting some weird things happening that me and the CI pipeline arent

raftmsohani commented:
> > > > i saw some failures testing locally, but i'm not sure what's related to the PR or not. it seems the datafile status endpoint returns a `502`, but looking at the logs, the only error i see is in the frontend
> > > > ```shell
> > > >    2025-12-12T15:15:53.51-0600 [APP/PROC/WEB/0] ERR 2025/12/12 21:15:53 [error] 118#0: *1977 limiting requests, excess: 1.000 by zone "limitreqsbyaddr", client: 2a09:bac1:76c0:f50::5e:27, server: , request: "POST /v1/logs/ HTTP/1.1", host: "tdp-frontend-raft.app.cloud.gov", referrer: "https://tdp-frontend-raft.app.cloud.gov/data-files?fy=2021&q=Q1&type=tanf&tab=1"
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > which is for a different request entirely and doesn't show up in the failures. in fact, searching for the specific request, i only see a success (reported in both backend and frontend logs)
> > > > ```
> > > > "GET /v1/data_files/954/ HTTP/1.1" 200 0 704
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > screenshot shows the different errors. some appear to be mismatched user permissions/locations for the test. i suppose the location IDs could be different in different envs, so maybe we need to `dumpdata --natural-foreign` like with user permissions. <img alt="Data file submission -- A FRA data analyst can submit a SSP data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079055-8a85d9d1-c551-45ff-a17b-2c728565abc7.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5MDU1LThhODVkOWQxLWM1NTEtNDVmZi1hMTdiLTJjNzI4NTY1YWJjNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZTZhN2Y4OWFjZmEyNGRkODFlMWVhOTY2OTZkNWRhYThmZDRkYzIzMzRiMzQ5Njc1ODIwZGQxOTFhYjY0NzUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FPLJKQOJMfFpNyJLqhxS34GiNf1z-zbLEipMqBZJgk4">
> > > > <img alt="Data file submission -- A FRA data analyst can submit a Work Outcomes of TANF Exiters data file (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079389-158ffb53-d84a-4137-acf4-554d1ce580ed.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5Mzg5LTE1OGZmYjUzLWQ4NGEtNDEzNy1hY2Y0LTU1NGQxY2U1ODBlZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjMwY2IyNDg4YzUyYjVlZTQ2M2YxZjhhMzQxMTFlYTY3MDczNTkwZTQzZTNhODE0MTU4NzFkZWQ1NGI1YWU3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oX1BbW2EsZwaHVquTHSddS5YWpnqB_Lz0mjv7Ao1dTI"> <img alt="Data file submission -- A FRA data analyst sees incorrect file type error (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526080293-b816e8e2-8363-42eb-9bcd-20030e772092.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDgwMjkzLWI4MTZlOGUyLTgzNjMtNDJlYi05YmNkLTIwMDMwZTc3MjA5Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMmZkNThkNTFjZTUzN2VjY2FjMTYyOGE2ZWU2MDNlYmRlZmRiMjIxYjNmNzkyMGM1YTk2Yzg0Y2Q5MjJjYWRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PSoqKg0GjL8OR8Abzb2eYUikF7cJvnrAvPuy04nKPAg"> <img alt="Data file submission -- Admin Alex can submit FRA reports (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079502-4b4a9c30-41f2-4e7e-887d-e4acecc3bce2.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTAyLTRiNGE5YzMwLTQxZjItNGU3ZS04ODdkLWU0YWNlY2MzYmNlMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYzBlMTAwOGZhYzFjNWJlMDU4NTg0NWMyNDUzMGYyZWE1MmIyODM5ZTM0YTUxZDhlM2JkZjAwNTFkMTEzZjFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FnZ_POcqnP_HFZZ04i54VGAb12Fg9sz6lv6WTNGCxZQ"> <img alt="Data file submission -- Data Analyst Stefani can submit a SSP Section 3 data file (example #7) (failed)" width="1280" height="633" src="https://private-user-images.githubusercontent.com/111305129/526079530-63e48e33-f458-45e8-91ea-d7894a9ac530.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU4MTA1ODMsIm5iZiI6MTc2NTgxMDI4MywicGF0aCI6Ii8xMTEzMDUxMjkvNTI2MDc5NTMwLTYzZTQ4ZTMzLWY0NTgtNDVlOC05MWVhLWQ3ODk0YTlhYzUzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxNVQxNDUxMjNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMDllMDY5ODFkNzJkNGNmYTM5ZWU5MjY4OTQzNzk1MmVmZTBmNTJjNTBiMzE0MWI4YmEzZWVkNjY4NThkNTA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KSlzjpKx1Z1Blx9pk68HOAlR1hBgSoC1S_wZmJZsDnA">
> > > 
> > > 
> > > @raftmsohani and @mattcoleanderson could you guys check and see if this is also happening for you?
> > 
> > 
> > @elipe17 I don't see any of the images and screenshots.
> 
> @raftmsohani are you able to successfully run the e2e test suite? Jan was getting some weird things happening that me and the CI pipeline arent

@elipe17 : no errors on my side

mattcoleanderson reviewed (approved):
Cannot wait to merge this!

elipe17 committed:
Merge branch 'develop' into 5495-referer-e2e-failures

elipe17 committed:
- Add longer timeout on intercept
- Move intercepts to the call immediately before the request
- Add extra check to ensure file submitted
